test/integration/services/alert.js

```diff
@@ -0,0 +1,89 @@
+describe('Alert Service', function() {
```

Comment:
this is my bad, but this should be `context`  it's an alias of `describe` but `describe`/`its` is for the `when this happens` tdd style tests, where as `context`/`specify` works better generally for our uses.